### DISCUSSION TITLE
webrtc demo cant display video on chrome  122.0.6261.58  version

### DISCUSSION THREAD

**Discussion by stoneinchina** [baresip/baresip#2933] [General]
baresip run  on linux machine ,then open 9000 port, after open web page, click connect,  then audio played, but video cant display.

Program run successfully before chrome version 122.0.6261.58

**Comment by sreimers** [comment]
Not reproducible here with latest chromium, can you provide the console log (browser + cli) and go to `chrome://webrtc-internals/` save the page after the test and upload here.

**Comment by stoneinchina** [comment]
when debug , the cli console have no error out, in the browser webrtc-internals plicount grow up all time. It looks like the video packet has been discarded by the browser all along
![webrtc_video_plicount](https://github.com/baresip/baresip/assets/42172039/7a902553-3458-42a5-976a-cf9f3cb70126)
[webrtc_internals_dump.txt](https://github.com/baresip/baresip/files/14416852/webrtc_internals_dump.txt)
[baresip_webrtc_cli_log.txt](https://github.com/baresip/baresip/files/14416949/baresip_webrtc_cli_log.txt)

**Comment by sreimers** [comment]
Is the demo code modified?

```
demo: request: met=GET, path=/, prm=
demo: loaded file './www/index.html', 4069 bytes (text/html)
demo: request: met=GET, path=/js/jquery-3.6.3.min.js, prm=
demo: loaded file './www/js/jquery-3.6.3.min.js', 89949 bytes (text/javascript)
demo: request: met=GET, path=/js/webrtc.js, prm=
demo: loaded file './www/js/webrtc.js', 13693 bytes (text/javascript)
demo: request: met=POST, path=/connect, prm=
demo: create session
demo: request: met=POST, path=/sdp, prm=
```

**Comment by stoneinchina** [comment]
yes, i changed some js code.  i also add some code in baresip to  input my  video and audio stream data. it works well on old chrome version. but chrome update last week, it cant work.

**Comment by sreimers** [comment]
Please try to reproduce with unchanged upstream code.

**Reply by stoneinchina** [reply]
[baresip_webrtc_log.txt](https://github.com/baresip/baresip/files/14475772/baresip_webrtc_log.txt)
![webrtc_baresip](https://github.com/baresip/baresip/assets/42172039/b19be426-7b4b-4138-aab3-7f88c7643725)
This is the unmodified original version of the log. In this version, I only added a module for inputting and outputting audio and video streams, called xsformat. In the latest version of Chrome browser, only audio with sound is visible, while the video is always loading. thanks
